core/primitives/src/stateless_validation/state_witness.rs

```diff
@@ -94,9 +94,6 @@ pub struct ChunkStateWitness {
     /// EpochId corresponds to the next block after chunk's previous block.
     /// This is effectively the output of EpochManager::get_epoch_id_from_prev_block
     /// with chunk_header.prev_block_hash().
-    /// This is needed to validate signature when the previous block is not yet
-    /// available on the validator side (aka orphan state witness).
-    // TODO(stateless_validation): Deprecate this field in the next version of the state witness.
```

Comment:
`epoch_id` is used in multiple places now, so this comment is no longer relevant